lms/djangoapps/discussion/django_comment_client/utils.py

```diff
@@ -146,6 +146,25 @@ def is_user_community_ta(user, course_id):
     return has_forum_access(user, course_id, FORUM_ROLE_COMMUNITY_TA)
 
 
+def get_users_with_moderator_roles(course_id):
+    """
+    Get all users within course with moderator roles
+    :param course_id:
+    :return:
+    """
+    moderators = {
+        user
+        for role in Role.objects.filter(
+            name__in=[FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR,
+                      FORUM_ROLE_COMMUNITY_TA, FORUM_ROLE_GROUP_MODERATOR],
```

Comment:
GROUP Moderator should only receive notifications for messages in their own group.